packages/flutter/test/material/date_picker_theme_test.dart

```diff
@@ -1206,4 +1205,67 @@ void main() {
       );
     }
   });
+
```

Comment:
Can you also add a test to verify that assigning `yearShape` works?

Reply:
Sure